docs/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+## Install
+
+```bash
+npm install leva
+```
+
+## Basics
+
+To use Leva, simply import `useControls` and use it anywhere in your app:
+
+```jsx
+import { useControls } from 'leva'
+
+function MyComponent() {
+  const { myValue } = useControls({ myValue: 10 })
+  return myValue
+}
+
+function AnotherComponent() {
+  const { anotherValue } = useControls({ anotherValue: "alive!!" })
+
+  return <div>Hey, I'm {anotherValue}</div>
+}
+
+function UnmountedComponent() {
+  const { barValue } = useControls({ barValue: false })
+
+  return barValue ? <div>Hello!</div> : null
+}
+
+function MyApp() {
+  
+  return (
+    <>
+      <MyComponent />
+      <AnotherComponent />
+    </>
+  )
+
+}
+```
+
+Note that since `UnmountedComponent` is not mounted anywhere in our application, the control declared there will not be added to the GUI!
+
+## Gotchas
+
+- The order of the controls depends on mounting order, in our case `myValue` will be first.
```

Comment:
Minor addition: mounting can be confusing. In fact I've made so that a parent control would show before its children's (I'm now storing the paths of the controls in `useMemo` which fires when the render **starts**).

```jsx
 import { useControls } from 'leva'

 function MyComponent() {
   const { myValue } = useControls({ myValue: 10 })
   return myValue
 }

 function AnotherComponent() {
   const { anotherValue } = useControls({ anotherValue: "alive!!" })

   return <div>Hey, I'm {anotherValue}</div>
 }

 function UnmountedComponent() {
   const { barValue } = useControls({ barValue: false })

   return barValue ? <div>Hello!</div> : null
 }

 function MyApp() {
   
   return (
       <MyComponent>
             <AnotherComponent />
       </MyComponent>
   )

 }
 ```

In that example, `myValue` will also be displayed first (when technically `MyComponent` finishes mounting **after** `AnotherComponent`.

Reply:
Call order would be better?

Reply:
Yes call order is indeed perfect!